x-pack/test/functional/apps/ml/anomaly_detection_jobs/types.ts

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
the file `date_nanos_job.ts` also uses these types.

Ideally we shouldn't need these types at all, and the test data should use the real ML types.
Looking at it quickly, they could be changed over, then `Detector` would need the `identifier` field removed. It looks like it is basically the description field.
I don't know if it's worth making these changes in this PR as it might a bit if churn.

Reply:
Updated here [`296a29d` (#160688)](https://github.com/elastic/kibana/pull/160688/commits/296a29da92af09ebe5813169740854b1cb25b90d)